src/sentry/runner/commands/backup.py

```diff
@@ -142,11 +149,24 @@ def scrub(self, on: InstanceID, left: JSONData, right: JSONData) -> None:
             left["scrubbed"] = {}
         if "scrubbed" not in right:
             right["scrubbed"] = {}
+
         for field in self.fields:
-            del left["fields"][field]
-            left["scrubbed"][f"{self.get_kind()}::{field}"] = True
-            del right["fields"][field]
-            right["scrubbed"][f"{self.get_kind()}::{field}"] = True
+            for side in [left, right]:
+                if field not in side["fields"]:
+                    continue
+                value = side["fields"][field]
+                if not value:
+                    continue
+                value = [value] if isinstance(value, str) else value
+                del side["fields"][field]
+                side["scrubbed"][f"{self.get_kind()}::{field}"] = f(value)
+
+    def scrub(
```

Comment:
> Public callers should use the inheritance-safe wrapper, `scrub`, rather than using this internal method directly.

Could you explain this more? I'm a bit confused why `__scrub__` is needed